aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,63 +1,14 @@
 ---
 remote_endpoint: ~
 proposals:
-  - name: enable_signature_checker
+  - name: upgrade_of_framework_v1_4_0_with_token_v2_hot_fix
     metadata:
-      title: "Enable the new signature checker"
-      description: "This enables the improved signature checker implementation."
+      title: "Fix the issue of wrong events emit from collection.move"
+      description: "This is a hot fix of framework v.1.4.0 to emit token v2 BurnEvent"
       source_code_url: "TBD"
-      discussion_url: "https://github.com/aptos-labs/aptos-core/pull/7856"
-    execution_mode: MultiStep
+      discussion_url: "https://github.com/aptos-labs/aptos-core/pull/8011"
+    execution_mode: SingleStep
```

Comment:
MultiStep, pretty much everything is multistep, maybe @runtian-zhou we can either make this default or error when this is not set properly

Reply:
Yeah, this is something I am not sure. It seems each subdir is a step even for single framework update.

Reply:
The name is misleading, we treat almost everything as "multistep" now

Reply:
I would encourage using multistep cuz we will be deprecate single step soon.